### ISSUE TITLE
Line breaks in messages are ignored

### ISSUE THREAD

**Issue by drygdryg** [verdigado/rocketchat2matrix#24]
Rocket.Chat uses newlines to separate lines in messages. Showdown by default expects Markdown-style line breaks (two spaces and a newline).
We should pass `simpleLineBreaks=true` ([doc](https://github.com/showdownjs/showdown/wiki/Showdown-options#simplelinebreaks)) to the Showdown options to convert the newline character to `<br>`.
https://github.com/verdigado/rocketchat2matrix/blob/56da775d0179391ac3dfd380b2944c7e3fdf7a3c/src/handlers/messages.ts#L100
```typescript
const converterOptions: showdown.ConverterOptions = { simpleLineBreaks: true }
```

**Comment by HerHde** [comment]
Thank you also for this addition, implemented in d0ea576e44676ff690d27e31df3a4e819c634edc

**Comment by HerHde** [comment]
Related to #29: _"Showdown parses differently than Element (and probably also different from RocketChat)"_
